openedx/adg/lms/registration_extension/tests/test_registration_form.py

```diff
@@ -0,0 +1,41 @@
+"""
+Tests for Registration form.
+"""
+
```

Comment:
Pls do not add blank line after module docs and method docs, Lets just remember this and we will fix this in some other ticket.